From 4bf063d8655698036c42ec34d6cc840d7bdc1a84 Mon Sep 17 00:00:00 2001
From: Jan Kiszka <jan.kiszka@siemens.com>
Date: Fri, 25 Nov 2016 17:31:51 +0100
Subject: [PATCH] ivshmem-net: Remove unused variable

Became unused by previous change.

Signed-off-by: Jan Kiszka <jan.kiszka@siemens.com>
(cherry picked from commit 3ea4b31deba3424784f0105c20dc90419e950e2c)
---
 drivers/net/ivshmem-net.c | 2 --
 1 file changed, 2 deletions(-)

--- a/drivers/net/ivshmem-net.c
+++ b/drivers/net/ivshmem-net.c
@@ -624,8 +624,6 @@ static irqreturn_t ivshm_net_int(int irq
 
 static int ivshm_net_open(struct net_device *ndev)
 {
-	struct ivshm_net *in = netdev_priv(ndev);
-
 	netdev_reset_queue(ndev);
 	ndev->operstate = IF_OPER_UP;
 	ivshm_net_run(ndev);
